content/sensu-go/5.12/installation/auth.md

```diff
@@ -1094,6 +1099,412 @@ example      | {{< highlight shell >}}"name": "activedirectory"{{< /highlight >}
+description  | Determines whether or not to skip SSL certificate verification (e.g. for self-signed certificates).
+required     | false
+type         | Boolean
+default      | false
+example      | {{< highlight shell >}}"insecure": true{{< /highlight >}}
+
+redirectURL  | 
+-------------|------
+description  | Redirect URL to provide to the OIDC provider. _NOTE: only required for certain OIDC providers, such as Okta._
+required     | false
+type         | String
+example      | {{< highlight shell >}}"redirectURL": "http://localhost:4000/login/callback"{{< /highlight >}}
+
+server       | 
+-------------|------
+description  | The location of the OIDC server you wish to authenticate against.
+required     | true
+type         | String
+example      | {{< highlight shell >}}"server": "https://localhost:9031"{{< /highlight >}}
+
+roles        | 
+-------------|------
+description  | An array of`roles` definitions.
+required     | true
+type         | Array
+example      | {{< highlight shell >}}"roles": [
+  {
+    "name": "guests",
+    "members": [
+      "myorganization/guests"
+    ],
+    "datacenters": [
+      "us-west-1"
+    ],
+    "subscriptions": [
+      "webserver"
+    ],
+    "readonly": true
+  },
+  {
+    "name": "operators",
+    "members": [
+      "myorganization/operators"
+    ],
+    "datacenters": [],
+    "subscriptions": [],
+    "readonly": false
+  }
+]
+{{< /highlight >}}
+
+#### `roles` attributes
+
+Please see the [RBAC definition specification][25] for information on how to
+configure RBAC roles.
+
+## Register an OIDC Application
+
+To use OIDC for authentication requires registration of your Sensu Enterprise
+Dashboard as an "application". Please note the following instructions to
+register an OIDC application for Sensu Enterprise based on your OIDC provider:
+
+- [Okta](#okta)
+- [PingFederate](#pingfederate)
+
+### Okta
+
+#### Requirements
+
+- Access to the Okta Administrator Dashboard
+- Sensu Enterprise Dashboard 2.9.0 or later
+
+#### Create an Okta Application
+
+1. From the Administrator Dashboard, select `Applications > Add Application > Create New App` to start the wizard.
+2. Select the `Web` platform and `OpenID Connect` sign in method.
+3. In General Settings enter an app name and (optionally) upload a logo.
+4. In Configure OpenID Connect, add the following Redirect URI, without forgetting to replace DASHBOARD_URL with the URL to your dashboard: `{DASHBOARD_URL}/login/callback`
+5. Click Save.
+6. Head over to the Sign On page and click on the Edit button of the OpenID Connect ID Token section.
+7. Enter the following information for the Groups claim attribute
+  - First field: `groups`
+  - Dropdown menu: `Regex`
+  - Second field: `.*`
+8. Click Save
+9. Make sure to assign people and/or groups in the Assignments page
+
+#### OIDC Driver Configuration
+
+1. Add the `additionalScopes` configuration attribute in the [OIDC scope][26] and set the value to `[ "groups" ]`, just like this:
```

Comment:
Instead now we would set `groupsClaim` to `groups` and `groupsPrefix` to whatever they want the associated group to show up as in RBAC.

For example, if my user on Okta is in group `dev` and we set the `groupsPrefix` to `okta:`, we'd setup the RBAC objects to mention group `okta:dev` as needed.